packages/lib/src/acting-io.ts

```diff
@@ -0,0 +1,170 @@
+import inquirer from 'inquirer'
+
+import { isIndexArgument } from './api-command'
+import { formatAndWriteItem, outputList, CommonListOutputProducer, CommonOutputProducer, ActionFunction, ListDataFunction,
+	Sorting, IdRetrievalFunction } from './basic-io'
+import { SmartThingsCommandInterface } from './smartthings-command'
+
+
+// Functions in this file support commands that act on an item. The biggest difference
+// between these and the methods from listing-io.ts is that they list items and immediately
+// query the user for an item to act on (if one wasn't specified on the command line). This
+// makes them safe to use for actions that make changes to the item or delete it.
+
```

Comment:
Commands in this file don't yet support the `acceptIndexId` implemented in the old version. I figured I would wait to implement that until I we translate a command that uses it which will give me a better feel of what a good API for it will be.

Reply:
Good idea...done.